opm/material/densead/Evaluation.hpp

```diff
@@ -255,9 +268,9 @@ class Evaluation
     Evaluation operator-() const
     {
         Evaluation result;
-        result.value_ = -this->value_;
-        for (unsigned varIdx = 0; varIdx < size; ++varIdx)
-            result.derivatives_[varIdx] = - this->derivatives_[varIdx];
+        // set value and derivatives to negative
```

Comment:
I added a free (non-class) operator- for FieldVector somewhere long ago (perhaps for CpGrid, don't remember), but that trick would not help much here. It is very annoying. I guess it is to avoid exposing temporary-creating operations, if I remember correctly?

However it prompts me to ask another thing: why not use std::array instead of Dune::FieldVector? It's an invisible internal detail and we don't gain much in terms of ease-of-writing operations for using FieldVector, it seems to me.